### PR TITLE
Create FloodFill op

### PR DESCRIPTION
**Pull request by lnyng**
see #391 

This branch create a wrapper op for the imglib2-algorithm.fill.FloodFill class. FillNamespace is also created for this op.

This PR is for review purpose. I feel that the `Filter<Pair<T, U>, Pair<T, U>>` and the `Writer<U>` are not looking very nice. I hope that they could be some kind of ops.

I am not sure if we also want some default `Filter`s. And do we need to write unit tests? since the op is just a wrapper.

### CODE REVIEW

**Comment by ctrueden** [comment]
Just a quick note: can't merge this until `imglib2-algorithm` is pinned to a release version.

**Comment by dietzc** [comment]
@tibuch any news on the integration of the `Imglib2-algorithm` `FloodFill`?

**Comment by tibuch** [comment]
I replaced the DefaultFloodFill with the imglib2 implementation. 

DefaultFloodFill is part of the morphology namespace and I haven't changed that. If you think a FillNamespace is needed, I can add it.

File: pom.xml [hunk 1]
```diff
@@ -204,6 +204,7 @@
 		<dependency>
 			<groupId>net.imglib2</groupId>
 			<artifactId>imglib2-algorithm</artifactId>
+			<version>0.5.1-SNAPSHOT</version>
```
Comment (dietzc) [inline pom.xml]: We have to wait until imglib2-algorithm was released or change this back if it has already been released.

**Comment by dietzc** [comment]
I don't think we need a namespace here. The existing namespace should be fine. I'm closing this issue. Feel free to open it again if you disagree.

**Pull request closed without merging**